R/nested_pipe_linter.R

```diff
@@ -76,7 +76,7 @@ nested_pipe_linter <- function(
     xml_nodes_to_lints(
       bad_expr,
       source_expression = source_expression,
-      lint_message = "Don't nest pipes inside other calls.",
+      lint_message = "Avoid nesting pipes inside other calls.",
```

Comment:
Consistency, although we could do with more:

```
$ grep -Fr '"Don\'t' R
R/T_and_F_symbol_linter.R:      make_lints(bad_assignment, "Don't use %2$s as a variable name, as it can break code relying on %2$s being %1$s.")
R/for_loop_index_linter.R:  lint_message = "Don't re-use any sequence symbols as the index symbol in a for loop."
R/condition_message_linter.R:      "Don't use", inner_call, "to build", outer_call, "strings.",
R/unnecessary_placeholder_linter.R:        "Don't use the placeholder (`.`) when it's not needed,",
R/unused_import_linter.R:        "Don't attach package '", unused_packages, "', which is only used by namespace. ",
R/package_hooks_linter.R:    lint_message <- sprintf("Don't use %s() in %s().", call_name, hook)
R/package_hooks_linter.R:      sprintf("Don't slow down package load by running installed.packages() in %s().", library_require_hook)
R/package_hooks_linter.R:      sprintf("Don't alter the search() path in %s() by calling %s().", library_require_hook, library_require_bad_call)
R/unnecessary_nested_if_linter.R:    "Don't use nested `if` statements,",

$ grep -Fr '"Avoid' R
R/assignment_linter.R:      "Avoid the assignment pipe %s; prefer using <- and %%>%% separately."
R/yoda_test_linter.R:      sprintf("Avoid storing placeholder tests like %s(1, 1)", matched_call)
R/pipe_return_linter.R:    "Avoid return() as the final step of a magrittr pipeline. ",
R/nested_pipe_linter.R:      lint_message = "Avoid nesting pipes inside other calls.",
R/nested_ifelse_linter.R:      sprintf("Avoid nested %s() calls.", matched_call),
R/implicit_integer_linter.R:        "Avoid implicit integers. Make the type explicit by using the form 1L for integers or 1.0 for doubles.",
R/one_call_pipe_linter.R:      lint_message = paste0("Avoid pipe ", pipe, " for expressions with only a single call."),
R/implicit_assignment_linter.R:      "Avoid implicit assignments in function calls.",
R/undesirable_operator_linter.R:    lint_message <- sprintf("Avoid undesirable operator `%s`.", operator)
```

Or do you think `Don't` and `Avoid` can both work?

Reply:
I think both can work. Don't feels a bit stronger, but I don't mind the variation.
Also note that in printed output, this consistency would create barrages of "Avoid" making it harder to distinguish lint types at a glance.

Reply:
OK, reverting to keep the diff smaller.